inc/saiqueue.h

```diff
@@ -258,6 +271,32 @@ typedef enum _sai_queue_stat_t
 } sai_queue_stat_t;
 
 /**
+ * @brief Enum defining Queue deadlock event state.
+ */
+typedef enum _sai_queue_pfc_deadlock_event_type_t
+{
+    /** PFC deadlock detected */
+    SAI_QUEUE_PFC_DL_START = 0x00000001,
```

Comment:
For this problem there are always 2 sides And each one claims they are right. Lets Bering this topic to sai meeting And see. As for is fomething id initialized or not, you write unittests once And you are set to go. You should write them im any of those cases